src/test/routes/single-or-multiple-claim.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from 'chai';
+import request from 'supertest';
+
+import { app } from '../../main/app';
+
+const expectedLegacyURL = 'https://employmenttribunals.service.gov.uk/apply';
```

Comment:
Please extract to a global constant

Reply:
Thanks Marc, I've added a contants.ts file within the main directory and included the URL.
https://github.com/hmcts/et-sya-frontend/blob/RET-508/src/main/constants.ts